src/data/apiClient.js

```diff
@@ -13,4 +16,69 @@ const apiClient = getAuthenticatedAPIClient({
   csrfCookieName: configuration.CSRF_COOKIE_NAME,
 });
 
+const clientServerKeyMap = {
+  bio: 'bio',
+  socialLinks: 'social_links',
+  country: 'country',
+  education: 'level_of_education',
+  fullName: 'name',
+  username: 'username',
+  profileImage: 'profile_image',
+  dateJoined: 'date_joined',
+  languageProficiencies: 'language_proficiencies',
+  accountPrivacy: 'account_privacy',
+};
+
+const serverClientKeyMap = _.invert(clientServerKeyMap);
+
+export function getPreferences(username) {
+  const url = `${lmsBaseUrl}/api/user/v1/preferences/${username}`;
+
+  return new Promise((resolve, reject) => {
+    apiClient.get(url)
+      .then(({ data }) => {
+        const createPathsAndTransformKeys = (acc, key) => {
+          _.set(
+            acc,
+            key.split('.').map(pathKey => serverClientKeyMap[pathKey] || pathKey),
+            data[key],
+          );
+          return acc;
+        };
+
+        const preferences = Object.keys(data).reduce(createPathsAndTransformKeys, {});
+
+        resolve(preferences);
+      })
+      .catch((error) => {
+        reject(error);
+      });
+  });
+}
+
+export function savePreferences(username, preferences) {
+  const url = `${lmsBaseUrl}/api/user/v1/preferences/${username}`;
+  // const data = {
+  //   "account_privacy": "custom",
+  //   "visibility.bio": "all_users",
+  //   "visibility.country": "private",
+  //   "visibility.language_proficiencies": "all_users"
+  // };
```

Comment:
Just checking that leaving this here is intentional.

Reply:
at the moment yet